t/api/02-iso.t

```diff
@@ -1130,4 +1130,42 @@ subtest 'circular reference' => sub {
         'circular reference');
 };
 
+subtest 'Test not trigger parent test suite' => sub {
```

Comment:
I would call this test `_SKIP_CHAINED_DEPS prevents scheduling parent tests`. This makes the difference of the test case more clear (which is usage of `_SKIP_CHAINED_DEPS`).